src/app/components/Modals/UnbondModal.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  Button,
+  Dialog,
+  DialogBody,
+  DialogFooter,
+  DialogHeader,
+  MobileDialog,
+  Text,
+} from "@babylonlabs-io/bbn-core-ui";
+import { useMediaQuery } from "usehooks-ts";
+
+import { screenBreakPoints } from "@/config/screen-breakpoints";
+
+interface UnbondModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+  onProceed: () => void;
+  awaitingWalletResponse: boolean;
+}
+
+export const UnbondModal = ({
+  isOpen,
+  onClose,
+  onProceed,
+  awaitingWalletResponse,
+}: UnbondModalProps) => {
+  const isMobileView = useMediaQuery(`(max-width: ${screenBreakPoints.md})`);
+
+  const DialogComponent = isMobileView ? MobileDialog : Dialog;
+
+  return (
+    <DialogComponent open={isOpen} onClose={onClose}>
+      <DialogHeader title="Unbonding" onClose={onClose} />
```

Comment:
We should probabably update the color to be `test-primary-dark` to match the design here as well.

<img width="604" alt="image" src="https://github.com/user-attachments/assets/02e646ec-2ada-4784-ab69-67f0dbfe2151">